packages/teleterm/src/ui/TopBar/Clusters/ClusterSelector/ClusterSelector.tsx

```diff
@@ -22,7 +22,13 @@ export const ClusterSelector = forwardRef<HTMLDivElement, ClusterSelectorProps>(
         isClusterSelected={!!props.clusterName}
         title={text}
       >
-        <Text css={{ whiteSpace: 'nowrap' }}>{text}</Text>
+        <Text
+          css={`
+            white-space: nowrap;
+          `}
```

Comment:
just curious between `css` and `style`, do both achieve the same thing (just syntax is a bit different?)

Reply:
@gzdunek Can you answer Lisa's question here? I remember you were changing that. ;)

Reply:
`css` works the same as `styled(Text)...` so you have access to the theme for example. Also, `css` generates a css class, while `style` adds inline style to the element.

Reply:
@gzdunek i see, thanks! should we be using `css` from now on? or does it matter?